test/integration.js

```diff
@@ -198,8 +198,8 @@ describe('integration tests:', function () {
       });
 
       it('should execute a long command with arguments', function (done) {
-        exec('very complicated deep command abc123 -rad -sleep \'well\' -t -i \'j\' ', done, function () {
-          stdout().should.equal('radtjabc123');
+        exec('very complicated deep command abc123 -rad --sleep \'well\' -t -i \'j\' ', done, function () {
```

Comment:
The test originally passed in `-sleep`, I thought it was a typo and changed it to `--sleep`. Is that correct?  Otherwise, we need to define options for 'l', 'e' and 'p'.  

Confirmed behavior of options with no [param] still being assigned the value in vorpal@1.5.5.